specification/msi/resource-manager/readme.java.md

```diff
@@ -0,0 +1,9 @@
+## Java
+
+These settings apply only when `--java` is specified on the command line.
+
+``` yaml $(java)
+add-client-flatten: Identity
```

Comment:
I didn't find the string in our codegen flag, not in m4 flag. It may not have effect.